opentelemetry-proto/src/proto/tonic/opentelemetry.proto.collector.logs.v1.rs

```diff
@@ -335,8 +331,11 @@ pub mod logs_service_server {
                         Ok(
                             http::Response::builder()
                                 .status(200)
-                                .header("grpc-status", "12")
-                                .header("content-type", "application/grpc")
+                                .header("grpc-status", tonic::Code::Unimplemented as i32)
+                                .header(
+                                    http::header::CONTENT_TYPE,
+                                    tonic::metadata::GRPC_CONTENT_TYPE,
+                                )
```

Comment:
@aumetra  - Just to be sure, are these files regenerated by running [grpc_build.rs](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-proto/tests/grpc_build.rs) against the new tonic version, and not manually modified ?

Reply:
Yeah, I ran the `cargo test` command to generate it using the tests